### ISSUE TITLE
Select styling duplicated

### ISSUE THREAD

**Issue by atmediauk** [techouse/select-auto-complete#3]
Hello,

Thanks for the package. I have a question though. When adding, it seems to be inserting multiple styling classes in to the generated select. For example I have the following:

```
            Select::make('Language', 'language_code')
                ->options(['en' => 'English', 'fr' => 'French', 'es' => 'Spanish'])
                ->displayUsingLabels(),
```

The select is generated and functions correctly but on the front end I see:
![image](https://user-images.githubusercontent.com/4824454/57449742-9d192180-7254-11e9-9df0-fa7ea459b2f6.png)

I have not added any custom select styling.

Thanks

**Comment by techouse** [comment]
I am not able to reproduce this issue with a vanilla Laravel Nova using this plugin and the code you pasted above.

![Works for me](http://i67.tinypic.com/242iq6u.jpg)

**Comment by atmediauk** [comment]
Hi @techouse 

Thanks for looking in to this. I've tried a few clean installs and keep getting the same result (with no console errors).

![image](https://user-images.githubusercontent.com/4824454/57616158-69503b80-7575-11e9-82cf-a65337cec34c.png)

I did notice if I delete this class ```form-select``` the field displays correctly.
 
![image](https://user-images.githubusercontent.com/4824454/57617592-0bbdee00-7579-11e9-94a8-91b9ff4de7a4.png)

Also I'm not sure if it helps but with my vanilla install I'm using the Laravel installer which installs the latest version (v5.8.16) and then pulling in the latest version of Nova via my composer file (which is v2.0.3). The Select Auto version is v1.0.4 in my lock file.

The only thing I then did was set up the database and add the language_code column. The other thing I did notice was even with the incorrect formatting I can still select an entry and save/ update it and it is stored in the database correctly.. 

![image](https://user-images.githubusercontent.com/4824454/57616810-0c558500-7577-11e9-8872-6637e9ac2d5e.png)

However when I look at the index view the language heading appears but the value remains empty.

![image](https://user-images.githubusercontent.com/4824454/57616899-4b83d600-7577-11e9-8ad7-09d8cb7cc1b5.png)

It also does not show the heading or value in show record view.

![image](https://user-images.githubusercontent.com/4824454/57617986-0f05a980-757a-11e9-9327-b99ca1341972.png)

Or pre-populates when going back in to the form to update.

Thanks for your help.

**Comment by techouse** [comment]
Interesting. Will take a deeper look into it when I have more time to spare. At the moment I'm really swamped. :/

**Comment by MsFoued** [comment]
i've got the same problem
You can fix it by deleting the css class specified in the capture

![tempsnip3](https://user-images.githubusercontent.com/15153313/57690060-193eab00-7641-11e9-8db5-28bbc3c638f7.png)

**Comment by techouse** [comment]
I had completely forgotten to [set the styling for the .card panel :D](https://github.com/techouse/select-auto-complete/commit/eef35427bef1d40a43625e4e98f4d8626c856447#diff-30e1f7a3a8f9f13f964f587ac452adcc)

It's been fixed in [v1.0.5](https://github.com/techouse/select-auto-complete/releases/tag/v1.0.5).

Thanks again to both of you for reporting.
